conversions/c_atoi_str_to_integer.c

```diff
@@ -0,0 +1,40 @@
+#include <stdio.h>
+
+// recoding the original atoi function in stdlib.h
+int	c_atoi(char *str)
+{
```

Comment:
please follow a indentation and formatting standard - LLVM is what we are standardizing throughout i.e.,
* no tabs, no spaces
* indent width = 4 spaces

Reply:
Done